recipes/reportlab/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "reportlab" %}
+{% set version = "3.4.0" %}
+{% set sha256 = "5beaf35e59dfd5ebd814fdefd76908292e818c982bd7332b5d347dfd2f01c343" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9   # [win and py27]
+    - vc14  # [win and py>=35]
```

Comment:
There are people fetching conda-forge recipes and building Python 3.4 packages.
So, if you add add the `vc10 ` here, you will make their lives easier.

However, do you need `vc`s for this package? If so please add them to `build` and `run` section below.
See http://conda-forge.github.io/docs/buildwin.html#vc-features